composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/polyfill-php84": "^1.30"
     },
     "require-dev": {
-        "phpunit/phpunit": "^11.5.41",
+        "phpunit/phpunit": "^12.3",
```

Comment:
That won't work on PHP 8.2, will it.

Reply:
but we can use it on the PHP 8.3/8.4 jobs